app.js

```diff
@@ -5,26 +5,30 @@ const eventKey = document.querySelector("#eventkey");
 const eventCode = document.querySelector("#eventcode");
 const eventWhich = document.querySelector("#eventwhich");
 
-document.addEventListener("keydown", () => {
+document.addEventListener("keydown", (event) => {
     console.log("interesting");
+    header.textContent = event.which;
+    eventKey.textContent = event.key;
+    eventCode.textContent = event.code;
+    eventWhich.textContent = event.which;
 })
 
-header.addEventListener("keydown", () => {
-    header.innerHTML = event.which;
-    // console.log(header);
-    // header.style.color = "red";
-})
+// header.addEventListener("keydown", (event) => {
+//     header.textContent = event.which;
+//     // console.log(header);
+//     // header.style.color = "red";
+// })
 
-eventKey.addEventListener("keydown", () => {
-    eventKey.textContent.innerHTML = event.key;
-})
+// eventKey.addEventListener("keydown", (event) => {
+//     eventKey.textContent = event.key;
+// })
 
-eventCode.addEventListener("keydown", () => {
-    eventCode.textContent = event.code;
-})
+// eventCode.addEventListener("keydown", (event) => {
+//     eventCode.textContent = event.code;
+// })
 
-eventWhich.addEventListener("keydown", () => {
-    eventWhich.textContent = keydown.which;
-})
+// eventWhich.addEventListener("keydown", (event) => {
+//     eventWhich.textContent = event.which;
+// })
```

Comment:
If you're commenting out code that already exists in git, you can just remove it, if you need it again you can get it back from the history :)